src/validation/fields/date.validation.ts

```diff
@@ -9,35 +9,47 @@ import {
 } from "../custom.validation";
 import { ErrorMessages } from "../error.messages";
 
+// to prevent more than 1 error reported on the date fields we check if the year is valid before doing some checks.
+// This means that teh year check is checked before some others
 export const start_date_validations = [
-  body("start_date")
+  body("start_date-day")
+    .if(body("start_date-year").isLength({ min: 4, max: 4 }))
```

Comment:
As discussed, this 'if' here is a bit different to how we check it in the custom functions, but agreed to leave as is for now.

It seems odd that we do the year check when validating the month field (line 19) but we don't also check the day...

Reply:
updated

Reply:
Not sure there was specifically an update for this. At least I can't see one and I thought we agreed to leave as is. Am I missing something?

Reply:
thought i updated everything, nothing to do for this one then